src/NClient.CodeGeneration/NClient.CodeGeneration.Abstractions/FacadeGenerationSettings.cs

```diff
@@ -6,15 +6,19 @@ public class FacadeGenerationSettings
     {
         public string Name { get; }
         public string Namespace { get; }
+        public bool GenerateClients { get; }
+        public bool GenerateFacades { get; }
         public bool UseModelValidationAttributes { get; }
         public bool UseNullableReferenceTypes { get; }
         public bool UseCancellationToken { get; }
         public bool UseDtoTypes { get; }
         public SerializeType SerializeType { get; }
         
         public FacadeGenerationSettings(
```

Comment:
Can we use records and init properties?

Reply:
Unfortunately not, because there is a framework in the targets
```C#
<TargetFrameworks>**net462;net472;net48;**net5.0;net6.0;netcoreapp3.1</TargetFrameworks>
```